.github/jira-mapping-rules.yml

```diff
@@ -0,0 +1,11 @@
+defaultJiraIssueType: 10009
+rules:
+  - type: label
+    labels:
+      - bug
+    jiraIssueType: 10011
+  - type: title
```

Comment:
can we remove this rule? features & pics should default as tasks too in JIRA so we don't lose visibility of them - we can always change them on the JIRA side if we needed to?

Reply:
Changed to map these as a Story instead, as noted they can be later changed into genuine JIRA epics if needed